### PR TITLE
Add OpenAPIv3 schema to CRDs

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>

### Description:

- Added hack/helm/schema/crd/main.go
    - The implementation is really simple. should be refactored later.
- Updated CRDs using this script.
- Also there's an update in hack/helm/schema/gen script.
    - now it can embed `description` into Helm schema JSON.

### Related Issue:

Nothing

### How Has This Been Tested?:

Nothing

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [X] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by rinx** [comment]
In my research, it is revealed that the object type sections without `properties` field needs `x-kubernetes-preserve-unknown-fields: true` attribute to work properly.

**Comment by rinx** [comment]
Also, it is needed to revise docs/user-guides/operations.md because Helm doesn't support upgrades of CRDs. If this PR is merged, the users of Helm Operator must upgrade the CRD manually.
https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

File: Makefile [hunk 1]
```diff
@@ -61,6 +61,7 @@ OPERATOR_SDK_VERSION := $(eval OPERATOR_SDK_VERSION := $(shell cat versions/OPER
 KIND_VERSION         ?= v0.10.0
 HELM_VERSION         ?= v3.5.2
 HELM_DOCS_VERSION    ?= 1.5.0
+YQ_VERSION           ?= v4.6.2
```
Comment (kpango) [inline Makefile:64]: sorry for reviewing late, it seems new version for YQ v4.6.3

**Review by kpango — CHANGES_REQUESTED** [review]
Others LGTM

File: Makefile (hunk 1 above)
Comment (rinx) [reply Makefile:64]: thanks for reviewing. 
```suggestion
YQ_VERSION           ?= v4.6.3
```

**Review by kpango — APPROVED** [review]
LGTM

**Pull request merged**